cuda/private/rules/cuda_library.bzl

```diff
@@ -104,7 +115,7 @@ cuda_library = rule(
         "hdrs": attr.label_list(allow_files = ALLOW_CUDA_HDRS),
         "deps": attr.label_list(providers = [[CcInfo], [CudaInfo]]),
         "alwayslink": attr.bool(default = False),
-        "rdc": attr.bool(default = False, doc = "whether to perform relocateable device code linking, otherwise, normal device link."),
+        "rdc": attr.bool(default = False, doc = "Whether to produce and consume relocateable device code. Transitive deps that contain device code must all either be cuda_objects or cuda_library that has rdc = True. If False, all device code must be in the same translation unit. May have performance implications. See https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#using-separate-compilation-in-cuda."),
```

Comment:
Could you please breakline a little bit, way too long for it.

Reply:
Done.